images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
@berrange Thanks for the clarification, but anyway we will need to hardcode sasl user and password for virt-manifest and virt-handler containers in the `auth.conf` file and also under saslpasswd2 command.
Can you give me some tips how can I avoid it?